src/Microsoft.Azure.WebJobs.Extensions.Storage/CloudStorageAccount/CloudStorageAccountBindingProvider.cs

```diff
@@ -19,19 +19,19 @@ public CloudStorageAccountBindingProvider(StorageAccountProvider accountProvider
             _accountProvider = accountProvider ?? throw new ArgumentNullException("accountProvider");
         }
 
-        public async Task<IBinding> TryCreateAsync(BindingProviderContext context)
+        public Task<IBinding> TryCreateAsync(BindingProviderContext context)
         {
             ParameterInfo parameter = context.Parameter;
 
             if (parameter.ParameterType != typeof(CloudStorageAccount))
             {
-                return null;
+                return Task.FromResult<IBinding>(null);
```

Comment:
Already fixed in latest: https://github.com/Azure/azure-webjobs-sdk/blob/3c3fc7dfa107bd65f1ea4e6d03580de66f404695/src/Microsoft.Azure.WebJobs.Extensions.Storage/CloudStorageAccount/CloudStorageAccountBindingProvider.cs#L33